app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">AndroidTrainning</string>
+    <string name="button">Button</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.button` appears to be unused
 Unused resources make applications larger and slow down builds.